tests/test_sandbox_recipe.py

```
@@ -0,0 +1,259 @@
+class TestLoadRecipe:
+    def test_load_valid_recipe(self, tmp_path: Path) -> None:
+        path = _write_recipe(tmp_path, VALID_RECIPE)
+        data = load_recipe(path)
+        assert data["schema_version"] == 1
+        assert data["validation_model"] == "docker"
+
+    def test_load_missing_file(self, tmp_path: Path) -> None:
+        path = tmp_path / "nonexistent.yml"
+        try:
+            load_recipe(path)
+            assert False, "Expected ValueError"
```

Ruff (0.15.15)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)